src/Service.Host/client/src/components/PurchaseOrders/PurchaseOrderUtility.js

```diff
@@ -98,6 +99,19 @@ function PurchaseOrderUtility({ creating }) {
         }
     }, [item, applicationState, creating, reduxDispatch]);
 
+    const [printHtml, setPrintHtml] = useState(<span>loading</span>);
+
+    useEffect(() => {
+        if (item?.orderNumber) {
+            fetch(config.appRoot + utilities.getHref(item, 'html'))
```

Comment:
config approot bit is so that it works locally as it needs to hit the backend but otherwise gets swallowed by localhost3000